### ISSUE TITLE
Team workspaces associated with the organization and not an individual

### ISSUE THREAD

**Issue by jkewley** [a85/POSTMan-Chrome-Extension#6100]
As an employee of a consulting firm which maintains collections in team workspaces across different clients, I'd like to see team workspaces be associated with the organization rather than an individual in the organization. Employees come and go, and having to worry about data loss during ownership transfer when reassigning licenses is not ideal.

I see personal workspaces as a place where I, as a dev, work on APIs and PM scripting. The team workspace is where the current, valid tests go - kind of like a gitflow approach where team workspaces are the master branch. 

To enable this flow, I want to be able to copy (not share) collections in my personal workspaces to team workspaces.

**Comment by abhijitkane** [comment]
@jkewley This can be accomplished by duplicating a collection in your personal workspace, and sharing the duplicate to the team workspace. This will prevent changes done on the duplicated collection (in the team workspace) from affecting your personal collection.

**Comment by jkewley** [comment]
@abhijitkane I can see how that would be a workaround for the private/shared workspace separation, but I hope you guys realize that's a workaround for a problem which needs to be resolved. We don't want to instruct our team members to double the size of their collections and/or workspaces in PM and then worry about which one is which as they work day to day.

As I think about it, since PM doesn't have a 'copy/move to collection' action, this would require devs to export the local collection and import it into the shared collection. Duplicate and drag/drop only works if you're using one giant personal workspace and not separating synced collections from local ones.

That also doesn't solve the problem of when a team member who was the lead dev for a particular client leaves the team and we have to reassign his license to his replacement. Some individual has to accept all of his collections and become the new owner as part of removing his account. The user who is assigned the collections has a ton of work to reog their 'My Workspace', and then they become the official owner. What if I wanted to divvy up his collections to different team members who have different relationships with clients? There is no reassign activity that I can see.

All this overhead, risk, and confusion about who owns what and where things are shared can be resolved if the team workspaces were simply owned by the org. That would let me fire all devs and revoke all licenses without having to worry about data loss in team workspaces. 

I hope I've been clear about my concerns

**Comment by preethammavin** [comment]
@jkewley I understand your concern, and owner concept is one which Postman as a Product wanted to move away from.
With Role Based access control post 7.0 version of the product, the creator of the collection is no longer the owner. This would mean any collection created in the team context is held with the team and doesn't go away when the creator leaves the team.
This is also something which we will be building upon for more fine-grain control across all Postman elements.


Also for the initial problem you mentioned, would suggest to use or fork and merge feature.Fork the collection from the team workspace to the personal workspace, continue editing when you are ready to merge, merge it with the collection in the team workspace.

**Comment by jkewley** [comment]
@preethammavin I'm glad that the team is aware of and is working on this issue. I tried creating a new collection in a team workspace and it still shows me as the owner:

![AddedToTeam](https://user-images.githubusercontent.com/1148930/54825644-72fba800-4c84-11e9-9751-28ece4fa2d6c.png)

Is this a bug in the UI, or would this collection be removed from the team workspace if my license was removed?

I'd strongly encourage the team to consider adding 'copy to collection' and/or 'move to collection' actions to the UI. This would greatly simplify organization of workspaces for end users.
